src/main/java/picard/fingerprint/ExtractFingerprint.java

```diff
@@ -103,7 +126,9 @@ protected int doWork() {
             checker.setDefaultSampleID(SAMPLE_ALIAS);
         }
 
-        final Map<String, Fingerprint> fingerprintMap = checker.identifyContaminant(INPUT.toPath(), CONTAMINATION);
+
+        final Map<String, Fingerprint> fingerprintMap = checker.identifyContaminant(INPUT.toPath(), CONTAMINATION, BACKGROUND == null ? null : BACKGROUND.toPath(),
+                BACKGROUND == null ? null : getCorrespondingSampleMap(INPUT, BACKGROUND), REMOVED_IS_TUMOR);
```

Comment:
what if the background and the input contain data with the same samplename? will this break?

Reply:
It should be fine.  Can add a test though to make sure.